src/vivarium_gates_bep/globals.py

```diff
@@ -41,4 +44,106 @@
     'neural_tube_defects': NEONATAL_CAUSE_LIST,
 }
 
+# disease names
```

Comment:
Consider using gbd mapping constants:
from gbd_mapping import causes
In [2]: causes.measles.name                                                                                             
Out[2]: 'measles'
In [3]: causes.meningitis.name                                                                                          
Out[3]: 'meningitis'

In [4]: causes.neonatal_preterm_birth.name                                                                              
Out[4]: 'neonatal_preterm_birth'